boxoffice/models/line_item.py

```diff
@@ -325,6 +334,11 @@ def get_confirmed_line_items(self):
 Order.get_confirmed_line_items = property(get_confirmed_line_items)
 
 
+def get_initial_line_items(self):
+    return LineItem.query.filter(LineItem.order == self, LineItem.previous == None, LineItem.status.in_(LINE_ITEM_STATUS.TRANSACTION))
+Order.get_initial_line_items = property(get_initial_line_items)
```

Comment:
If it's a property it shouldn't have the `get_` prefix. It'll be used as `order.initial_line_items.all()`, similar to how a lazy relationship will be.